jest.config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
   preset: 'ts-jest',
   testEnvironment: 'node',
+  modulePathIgnorePatterns: ['<rootDir>/.test-space/*']
```

Comment:
needed to add this otherwise the tests infinitely loop when watching as it would write the files for the test project, trigger another test run, then write the files again, trigger, etc.